docs/traceability/index.md

```diff
@@ -958,18 +958,21 @@ The Agent SDK provides support for specifying the version of the agent at the bu
 - github.com/Axway/agent-sdk/pkg/cmd.BuildVersion
 - github.com/Axway/agent-sdk/pkg/cmd.BuildCommitSha
 - github.com/Axway/agent-sdk/pkg/cmd.BuildAgentName
+- github.com/Axway/agent-sdk/pkg/cmd.SDKBuildVersion
 
 The following is an example of the build command that can be configured in the Makefile
 
 ```
 @export time=`date +%Y%m%d%H%M%S` && \
 export version=`cat version` && \
 export commit_id=`git rev-parse --short HEAD` && \
+export sdk_version=`go list -m github.com/Axway/agent-sdk | awk '{print $$2}' | awk -F'-' '{print substr($$1, 2)}'` && \
 go build -tags static_all \
 	-ldflags="-X 'github.com/Axway/agent-sdk/pkg/cmd.BuildTime=$${time}' \
 			-X 'github.com/Axway/agent-sdk/pkg/cmd.BuildVersion=$${version}' \
 			-X 'github.com/Axway/agent-sdk/pkg/cmd.BuildCommitSha=$${commit_id}' \
 			-X 'github.com/Axway/agent-sdk/pkg/cmd.BuildAgentName=SampleTraceabilityAgent'" \
+			-X 'github.com/Axway/agent-sdk/pkg/cmd.SDKBuildVersion=$${sdk_version}' \
```

Comment:
anyone else bothered, a little, that the other variables are cmd.Build... and the new one is cmd.SDKBuild...

Reply:
Now that you mention it

Reply:
I thought about that. But in a noun/verb situation, if you were to ask it in a sentence, one would say "what is the build version", or "what is the SDK build version". Not "what is the build sdk version". So, in my little world I would now change it to "AgentBuildVersion" also, since there are 2 build versions.

But, I digress... not a big deal. I can change them all if you want.

Reply:
This is fine, not worth the change.